tests/unit/tests/ocf_env/ocf_env_headers.h

```diff
@@ -1,13 +1,28 @@
 /*
- * Copyright(c) 2012-2018 Intel Corporation
+ * Copyright(c) 2019 Intel Corporation
  * SPDX-License-Identifier: BSD-3-Clause-Clear
  */
 
+/**
+ * @file
+ * @brief OCF libs and macros
+ *
+* This file is mostly to aggregate external header includes.
+ */
 #ifndef __OCF_ENV_HEADERS_H__
 #define __OCF_ENV_HEADERS_H__
 
 #include <stdint.h>
-#include <stdbool.h>
 #include <stddef.h>
+#include <stdbool.h>
+
+/* TODO: Move prefix printing to context logger. */
+#define OCF_LOGO "Intel(R) OCF"
```

Comment:
We shouldn't use "Intel" branding anywhere in Open CAS * projects

Reply:
You're right, thanks.